content/docs/self-hosted-runners.md

```diff
@@ -73,10 +76,48 @@ jobs:
           pip install -r requirements.txt
           python train.py
 
+          # Create CML report
           cat metrics.txt >> report.md
+          cml publish plot.png --md >> report.md
           cml send-comment report.md
 ```
 
+</tab>
+<tab title="GitLab">
+
+```yaml
+deploy-runner:
+  image: iterativeai/cml:0-dvc2-base1
+  script:
+    - |
+      cml runner \
+          --cloud=aws \
+          --cloud-region=us-west \
+          --cloud-type=p2.xlarge \
+          --cloud-spot \
+          --labels=cml-gpu
```

Comment:
Not a fan of this barely idiomatic line continuation syntax, but... 🤷🏼‍♂️

Reply:
if you can configure the linter to accept any other reasonable format please do so :)

Reply:
`prettier --prose-wrap=preserve` or something similar? 🤔

Reply:
Still, it's a bit too broad.

Reply:
Cross–linking to https://github.com/iterative/cml/pull/779#issuecomment-952140244 for future references.